package.json

```diff
@@ -13,19 +13,30 @@
     "polymer-bundler": "lib/bin/polymer-bundler.js"
   },
   "dependencies": {
+    "@types/acorn": "^4.0.3",
+    "@types/babel-generator": "^6.25.1",
+    "@types/babel-traverse": "^6.25.3",
+    "@types/babylon": "^6.16.2",
+    "@types/rollup": "^0.54.0",
+    "babel-generator": "^6.26.1",
+    "babel-traverse": "^6.26.0",
+    "babylon": "^6.18.0",
     "clone": "^2.1.0",
     "command-line-args": "^3.0.1",
     "command-line-usage": "^3.0.3",
     "dom5": "^2.2.0",
     "espree": "^3.5.2",
+    "magic-string": "^0.22.4",
     "mkdirp": "^0.5.1",
     "parse5": "^2.2.2",
-    "polymer-analyzer": "^3.0.0-pre.10",
+    "polymer-analyzer": "^3.0.0-pre.13",
+    "rollup": "^0.56.1",
     "source-map": "^0.5.6",
     "vscode-uri": "^1.0.1"
   },
   "devDependencies": {
     "@types/chai": "^3.4.30",
+    "@types/chokidar": "^1.7.5",
```

Comment:
TypeScript compilation blows up because one of the chokidar types is referenced in an options hash inside rollup (I think it was rollup.)  Anyways, its annoying.